### PR TITLE
*: miscellaneous updates on release 3.2 cycle

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

**Comment by xiang90** [comment]
CI is failing.

File: MAINTAINERS [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Anthony Romano <anthony.romano@coreos.com> (@heyitsanthony) pkg:*
 Brandon Philips <brandon.philips@coreos.com> (@philips) pkg:*
+Fanmin Shi <fanmin.shi@coreos.com> (@fanminshi) pkg:*
```
Comment (fanminshi) [inline MAINTAINERS]: Nice!

File: test [hunk 2]
```diff
@@ -9,7 +9,7 @@
 # PKG=./wal ./test
 # PKG=snap ./test
 #
-# Run code coverage
```
Comment (fanminshi) [inline test]: do you have auto removing trailing space on for your IDE?

Comment (gyuho) [inline test]: My Vim just highlights those trailing spaces :)

**Comment by gyuho** [comment]
@xiang90 @fanminshi All fixed. PTAL. Thanks.

**Pull request merged**